Themes/default/Errors.template.php

```diff
@@ -225,7 +225,7 @@ function template_show_file()
 		$is_target = $line_num == $context['file_data']['target'];
 		echo '
 			<tr>
-				<td align="right"', $is_target ? ' class="current">==&gt;' : '>', $line_num , ':</td>
+				<td ', $is_target ? ' class="current righttext" : "righttext">==&gt;' : '>', $line_num , ':</td>
```

Comment:
<td class="righttext', $is_target ? ' current">==&gt;' : '">' is cleaner. Considering its already has align right righttext needs to be there as well.